docs/usage/help.rst.inc

```diff
@@ -58,8 +58,16 @@ Regular expressions, selector `re:`
     the re module <https://docs.python.org/3/library/re.html>`_.
 
 Path prefix, selector `pp:`
-    This pattern style is useful to match whole sub-directories. The pattern
-    `pp:root/somedir` matches `root/somedir` and everything therein.
+    This pattern style is useful to match whole sub-directories without worrying
+    about escaping special characters. Otherwise, you can just use the default
+    `fm:` or `sh:` styles, which also match whole sub-directories. Unlike those
+    styles, a trailing slash here has no effect on what gets matched. The pattern
+    `pp:root/somedir` (and `pp:root/somedir/`) matches `root/somedir` and
+    everything therein.
+
+    When using exclude-norecurse rules, this style is equivalent to `pf:`,
+    except that `pf:` won't match patterns with trailing slashes (and `pp:`
```

Comment:
similar here, please describe the specialities of `pf` in the pf description, see below.

Reply:
It's comparing `pf` and `pp`, so it seems like either way, one of the sections will be referring to the other.

Reply:
that pf: needs to be used without trailing slashes (when specifying dirs) should be documented in the pf section.

Reply:
(and we could change the code to strip any trailing slashes for pf:)